devito/__init__.py

```diff
@@ -0,0 +1,3 @@
+class NamedObject(object):
+    def getName(self):
```

Comment:
Why are we creating an extra class here and not just adding this to `Operator` itself? Also, why is this supposed to live in `__init__.py`?